docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json

```diff
@@ -0,0 +1,345 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://azconfig.io/schemas/FeatureManagement/v2.0.0/FeatureFlag.json",
+  "type": "object",
+  "title": "An Azure App Configuration Feature Declaration",
+  "required": [
+    "id"
+  ],
+  "properties": {
+    "id": {
+      "$id": "#/properties/id",
+      "type": "string",
+      "title": "Feature ID",
+      "description": "An ID used to uniquely identify and reference the feature.",
+      "examples": [
+        "fancy-background"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "description": {
+      "$id": "#/properties/description",
+      "type": "string",
+      "title": "Feature Description",
+      "description": "A description of the feature.",
+      "default": "",
+      "examples": [
+        "This will display a fancy background on the home page."
+      ],
+      "pattern": "^[a-zA-Z0-9_\\-\\.]*$"
+    },
+    "display_name": {
+      "$id": "#/properties/display_name",
+      "type": "string",
+      "title": "Feature Display Name",
+      "description": "A display name for the feature to use for display rather than the ID.",
+      "default": "",
+      "examples": [
+        "Fancy Background"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "enabled": {
+      "$id": "#/properties/enabled",
+      "type": "boolean",
+      "title": "Enabled State",
+      "description": "A feature is OFF if enabled is false. If enabled is true, then the feature is ON if there are no conditions (null or empty) or if the conditions are satisfied.",
+      "default": false
+    },
+    "conditions": {
+      "$id": "#/properties/conditions",
+      "type": "object",
+      "title": "Feature Enablement Conditions",
+      "description": "The declaration of conditions used to dynamically enable features.",
+      "required": [],
+      "properties": {
+        "requirement_type": {
+          "$id": "#/properties/conditions/properties/requirement_type",
+          "type": "string",
+          "title": "Requirement Type",
+          "description": "Determines whether any or all registered client filters must be enabled for the feature to be considered enabled.",
+          "enum": [
+            "Any",
+            "All"
+          ],
+          "default": "Any"
+        },
+        "client_filters": {
+          "$id": "#/properties/conditions/properties/client_filters",
+          "type": "array",
+          "title": "Client Filter Collection",
+          "description": "Filters that must run on the client and be evaluated as true for the feature to be considered enabled.",
+          "items": {
+            "$id": "#/properties/conditions/properties/client_filters/items",
+            "type": "object",
+            "title": "Client Filter",
```

Comment:
I saw this in the v1.1.0 file and I interpreted it as a singular filter because it's describing how each individual filter in the list should look, and there's already a `Client Filter Collection` object.